neps/nep-0568.md

```diff
@@ -112,6 +112,20 @@ Cold storage uses the same mapping strategy to manage shard state during reshard
 
 This approach minimizes complexity while maintaining consistency across hot and cold storage.
 
+#### State cleanup
+
+Since [Stateless Validation][NEP-508], all shards tracking is no longer required. Currently, shard cleanup has not been implemented. We propose a cleanup mechanism that will also handle post-resharding cleanup.
+
```

Comment:
Can you define briefly what is state cleanup and under what circumstances it is needed?

Reply:
fixed in 333544c7499f10b61773c81779f4c8116984ebd5